omim2obo/mondo_omim_genes_robot_tsv.py

```diff
@@ -0,0 +1,64 @@
+"""Create: ROBOT template of Mondo and OMIM gene relations: relational information for gene and disease classes"""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, Union
+
+import pandas as pd
+
+from omim2obo.utils.utils import remove_angle_brackets
+
+
+ROBOT_SUBHEADER = {
+    'mondo_id': 'ID',
+    'hgnc_id': "SC 'has material basis in germline mutation in' some %",
+    'omim_disease_xref': '>A oboInOwl:source',
+    'omim_gene': '',
+}
+
+
+def mondo_omim_genes_robot_tsv(inpath: Union[Path, str], outpath: Union[Path, str]) -> pd.DataFrame:
+    """Create: ROBOT template of Mondo and OMIM gene relations"""
+    df = pd.read_csv(inpath, sep='\t')
+
+    # Remove the first character, a question mark (?), from each field in the header; an artefact of the SPARQL query.
+    df.rename(columns={col: col[1:] for col in df.columns if col.startswith('?')}, inplace=True)
+
+    # Remove < and > characters from specified columns
+    uri_cols = ['mondo_id', 'hgnc_id', 'omim_gene']
+    for col in uri_cols:
+        df[col] = remove_angle_brackets(list(df[col]))
+
+    # Format col order
+    df = df[['mondo_id', 'hgnc_id', 'omim_disease_xref', 'omim_gene']]
+
+    # Sort
+    df = df.sort_values(by=['mondo_id', 'hgnc_id', 'omim_gene', 'omim_disease_xref'])
+
+    # Remove cases where >1 gene association
+    # - These indicate non-causal relationships, which we don't care about.
+    df = df[~df['omim_disease_xref'].duplicated(keep=False)]
```

Comment:
## Remove cases where >1 gene association
@twhetzel @matentzn Done.

Apologies, normally I should upload a google sheet, but don't have time for that right now. Tomorrow I'll update the release with this new version of the file, then I'll update `mondo-ingest` as well, committing that file.

[mondo-omim-genes.robot.tsv.zip](https://github.com/user-attachments/files/15893334/mondo-omim-genes.robot.tsv.zip)

Reply:
FYI - please check with Sabrina before doing a build in `mondo-ingest`

Reply:
No build; just simply committing the update to the robot template and its corresponding OWL; but I meant in [the PR](https://github.com/monarch-initiative/mondo-ingest/pull/559), not on `main`. And @matentzn can perhaps update his `mondo` PR by pulling that file directly, rather than from `main`.

Reply:
Sounds good. I'm just going to update the `mondo-ingest` PR as well, as that one also needs to be merged and is now out of date:
- https://github.com/monarch-initiative/mondo-ingest/pull/559

This PR has been ready as of yesterday, feel free to review and we can merge.

- [ ] @twhetzel and @matentzn re-approve